app/Http/Controllers/MoviesController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+//use App\Http\Requests\EditRequest;
+use App\Http\Requests\Movies\EditRequest;
+use App\Http\Requests\Movies\MoviesRequest;
+//use App\Http\Requests\MoviesRequest;
```

Comment:
закоментированный код удаляем
```suggestion
use App\Http\Requests\Movies\EditRequest;
use App\Http\Requests\Movies\MoviesRequest;
```